scripts/preprocessing/03-make_epochs.py

```diff
@@ -61,7 +72,19 @@ def run_epochs(subject, session=None):
     else:
         raw = mne.concatenate_raws(raw_list)
 
-    events, event_id = mne.events_from_annotations(raw)
+    #Events for rest session
+    if config.session == 'rest' :
```

Comment:
```suggestion
    if config.task == 'rest' :
```

it's the task that says it's rest.